src/annotator/guest.js

```diff
@@ -121,14 +121,13 @@ export default class Guest extends Delegator {
         this.selectAnnotations(anns);
       },
     });
-    this.selections = selections(document).subscribe({
-      next: range => {
-        if (range) {
-          this._onSelection(range);
-        } else {
-          this._onClearSelection();
-        }
-      },
+
+    this.selectionObserver = new SelectionObserver(range => {
```

Comment:
It's a nice little detail that the API surface is consistent in the new observer class, but without the extra layer of "subscribing".

Reply:
Yeah, using `Observable` seemed like a nice idea when the API appeared to be on track to be standardized as part of ES. However progress on that stalled.